docs/selfhealing.md

```diff
@@ -0,0 +1,87 @@
+# Self Healing
+ALBA has built-in self healing functionality. In case an ASD is broken, fragments on the ASD will be automatically reconstructed on remaining disks in the Backend.
+
+The self healing work consists out of several types of maintenance work:
```

Comment:
Not all of the types of maintenance work listed below are related to self healing